config/default.yml

```diff
@@ -4632,6 +4632,11 @@ Style/TernaryParentheses:
     - require_parentheses_when_complex
   AllowSafeAssignment: true
 
+Style/TopLevelMethodDefinitions:
```

Comment:
Let's use singular here (Definition) to be aligned with how most of the cops are named.